acquire/dynamic/windows/ntdll.py

```diff
@@ -29,6 +31,17 @@
 
 
 ntdll = ctypes.windll.ntdll
+ntdll.NtQueryInformationFile.argtypes = (
```

Comment:
I get the idea of not putting `NtQueryInformationFile` in this module's namespace, but for the sake of consistency it maybe should, like `NtOpenDirectoryObject` and others below.